src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,129 @@
+package frc.robot.subsystems.arm;
+
+//importing libraries
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+// extend SubsystemBase class into the Arm class?
+public class Arm {
+
+    // making variable under the CANSparkMax class
+    private CANSparkMax leftMotor;
+    private CANSparkMax rightMotor;
+
+    //variables to set limits, speeds, and ramp-rate
+    private final float REVERSE_LIMIT = (float) 0;
+    private final float FORWARD_LIMIT = (float) 0;
+    private final double REVERSE_SPEED = 0;
+    private final double FORWARD_SPEED = 0;
+    private final double RAMPRATE = 0;
+
+    //variables to set the arm position when scoring and intaking
```

Comment:
These variables will not do what you want.

What you need instead is a map of armPosition enums to the value you pass to the arm motor controller to set the postion.  

Create some floats (they may need to be doubles) such as:
private final float INTAKE_ARM_GROUND = 0.0f;
...

Then, add a map of the enum to the float value:
Map<armPosition, float> armPositions = new Map<armPosition, float>();
armPositions .put(armPosition.INTAKE_ARM_POSITION_GROUND, INTAKE_ARM_GROUND);
...

You can delete the 4  armPositions you have here, and just make one called currentArmPosition.  

Then, in each runTo[action][position] method, set currentArmPosition.

Last, when it comes time  to drive the arm itself, you will use the map to figure out where to go.
when you call armPositions.get(currentArmPosition) it will give you the value of where to put the arm.